Commands.py

```diff
@@ -0,0 +1,19 @@
+
+userInput = input()
+
+#run function that displays all the commands the user can type
+if(userInput == "!help"):
+    #make prettier later, I forget how to have a print statement span multiple lines of code.
+    print('''
+!newpuzzle: This command generates a new puzzle. If you are generating a puzzle from a chosen pangram, enter said pangram along with the command.
```

Comment:
I think you can get rid of "This command" for all of them here since it's redundant! "Generates a new puzzle...." etc.